ansibullbot/_text_compat.py

```diff
@@ -0,0 +1,278 @@
+# This code is part of Ansible, but is an independent component.
```

Comment:
Since ansible is a [dependency](https://github.com/ansible/ansibullbot/blob/1e459530cc262f088b80140289fbb5dae7087c85/ansibullbot/utils/component_tools.py#L168) of ansibullbot, could not ansibullbot directly import Ansible methods ? Maybe these methods should be moved in a dedicated Python package ? Same comment for `constants.py` file.

Reply:
It's a non-direct dependency, which resides in a separate env. So no. Also, I'd not want to make these two tightly coupled.